### PR TITLE
document using options per command

### PR DESCRIPTION
**Pull request by mojavelinux**
- document that options can be added to a command
- document that command options are validated when the command is used
- document that options are not validated if command has no action

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -64,6 +64,27 @@ if (program.sauce) console.log('  with sauce');
 else console.log(' without sauce');
 ```
 
+## Options for a command
+
+You can attach options to an action-based command.
+
+```ruby
```
Comment (abetomo) [inline Readme.md]: javascript?

Comment (mojavelinux) [reply Readme.md]: OMG! I'm having a polyglot moment!

Comment (mojavelinux) [reply Readme.md]: Fixed!

Comment (abetomo) [reply Readme.md]: Thank you!

**Pull request merged**